sway-core/src/control_flow_analysis/dead_code_analysis.rs

```diff
@@ -1462,6 +1488,20 @@ fn connect_expression<'eng: 'cfg, 'cfg>(
             }
             Ok(vec![node])
         }
+        ConfigurableExpression {
+            decl: const_decl, ..
+        } => {
+            let node = if let Some(node) = graph.namespace.get_configurable(const_decl) {
+                *node
+            } else {
+                return Ok(leaves.to_vec());
+            };
```

Comment:
This looks like it could just be a let-else.

Reply:
Done.